docs/_plugins/importMap.cjs

```diff
@@ -2,33 +2,47 @@
 const { join } = require('node:path');
 const { pathToFileURL } = require('node:url');
 const { glob } = require('glob');
+const { AssetCache } = require('@11ty/eleventy-fetch');
 
-module.exports = function(eleventyConfig, {
-  inputMap = undefined,
-  defaultProvider = undefined,
-  localPackages = [],
-} = {}) {
-  const cwd = process.cwd();
-  const elementsDir = join(cwd, 'elements/');
-
-  const specs = localPackages.map(spec => ({
-    spec,
-    packageName: spec.replace(/^@/, '$').replace(/@.*$/, '').replace(/^\$/, '@')
-  }));
-
-  // copy over local packages
-  for (const { packageName } of specs) {
-    eleventyConfig.addPassthroughCopy({ [`node_modules/${packageName}`]: `/assets/packages/${packageName}` });
-  }
-
-  // HACK: copy lit transitive deps
-  // this might not be necessary if we flatten to a single lit version
-  for (const packageName of ['lit-html', 'lit-element']) {
-    eleventyConfig.addPassthroughCopy({ [`node_modules/${packageName}`]: `/assets/packages/${packageName}` });
+function logPerf() {
+  // We should log performance regressions
+  /* eslint-disable no-console */
+  const chalk = require('chalk');
+  const TOTAL = performance.measure('importMap-total', 'importMap-start', 'importMap-end');
+  const RESOLVE = performance.measure('importMap-resolve', 'importMap-start', 'importMap-afterLocalPackages');
+  const TRACE = performance.measure('importMap-trace', 'importMap-afterLocalPackages', 'importMap-afterRHDSTraces');
+  if (TOTAL.duration > 2000) {
+    console.log(
+      `🦥 Import map generator done in ${chalk.red(TOTAL.duration)}ms\n`,
+      `  Resolving local packages took ${chalk.red(RESOLVE.duration)}ms\n`,
+      `  Tracing RHDS sources took ${chalk.red(TRACE.duration)}ms`,
+    );
+  } else if (TOTAL.duration > 1000) {
+    console.log(
+      `🐢 Import map generator done in ${chalk.yellow(TOTAL.duration)}ms\n`,
+      `  Resolving local packages took ${chalk.yellow(RESOLVE.duration)}ms\n`,
+      `  Tracing RHDS sources took ${chalk.yellow(TRACE.duration)}ms`,
+    );
+  } else {
+    console.log(
+      `⚡ Import map generator done in ${chalk.blue(TOTAL.duration)}ms\n`,
+    );
```

Comment:
Would be nice in the future on ci build in github if we could ship these somewhere to store and then pull back to generate a graph on performance over time as well.

Reply:
It should be straightforward (hah!) To cache the outputs against a hash of the dependencies (currently this file and the eleventy config) and restore the flat cache on each run